test/data/README.md

```diff
@@ -65,6 +65,12 @@ The dataset is under a CC-by 4.0 license.
 }
 ```
 
+### Boston Housing Data
```

Comment:
Can you add a more info link? I also added a link to our copy of the dataset so folks know what we are referencing. 
```
### Boston Housing Data
 Redistributing the dataset "[housing.txt](housing.txt)" with attribution:
 > Harrison, D. and Rubinfeld, D.L. 'Hedonic prices and the demand for clean air', J. Environ. Economics & Management, vol.5, 81-102, 1978.

More information: http://b-course.hiit.fi/obc/housingexpl.html
```

Reply:
Done.